zerver/lib/upload/s3.py

```diff
@@ -145,6 +145,10 @@ class S3UploadBackend(ZulipUploadBackend):
     def __init__(self) -> None:
         self.avatar_bucket = get_bucket(settings.S3_AVATAR_BUCKET)
         self.uploads_bucket = get_bucket(settings.S3_AUTH_UPLOADS_BUCKET)
+        self.export_bucket: Bucket | None = None
+        if settings.S3_EXPORT_BUCKET:
+            self.export_bucket = get_bucket(settings.S3_EXPORT_BUCKET)
```

Comment:
Would it work to just have `export_bucket` equal `uploads_bucket` rather than `None` if `S3_EXPORT_BUCKET` is not set? I feel like that might avoid a big conditional duplicating code below, but maybe I'm missing something.

Reply:
I guess?  I liked the cleanliness of the path without `exports` in it twice.

Reply:
OK. We can always refactor the internals of this later if it seems warranted.